corehq/apps/callcenter/views.py

```diff
@@ -40,6 +41,7 @@ def static_options(self):
         return []
 
 
+@location_safe
```

Comment:
What's this for?  Is this widget used on a location-safe page?  It doesn't hurt to add as long as it's restriced by location, of course, just didn't recognize this as part of a page/workflow I knew to be location safe.